tests/integration/suite/test_notifier.py

```diff
@@ -0,0 +1,21 @@
+from .common import random_str
+
+
+def test_notifier_wechat_apiurl(admin_mc, remove_resource):
```

Comment:
What is being tested? Shouldn't we attempt the "send" action? Otherwise, I don't think the added logic is being tested by this.

Reply:
I assume we should at least perform the action on two different inputs:
1. using a blank url- I would expect this would return as a success.
2. a url like test.com- I would expect this would return an error.

Reply:
It needs proper tokens in order to get a successful test, and we don't want to include those in the tests. Does it make sense to you if we do a test to the default URL, and check an expected error?

Reply:
@XianglongLuo maybe you can try to add a unit test and mock `http` for it